build.gradle

```diff
@@ -256,29 +256,27 @@ def syncFrontendPublic = tasks.register('syncFrontendPublic', Sync) {
     }
 }
 
-def macHotReloadFrontend = tasks.register('macHotReloadFrontend', Exec) {
-    dependsOn installFrontend
+def macHotReloadFrontend = tasks.register('macHotReloadFrontend', com.liferay.gradle.plugins.node.tasks.ExecutePackageManagerTask) {
     onlyIf {os.isMacOsX()}
     workingDir 'frontend/'
-    commandLine 'npm', 'run', 'serveOpen'
+    args = ['run', 'serveOpen']
 }
 
 def windowsHotReloadFrontend = tasks.register('windowsHotReloadFrontend', Exec) {
```

Comment:
Can I double check again on this, do we need to change the `Exec` to `com.liferay.gradle.plugins.node.tasks.ExecutePackageManagerTask`?

Reply:
I think it has already been changed? It was changed to be consistent with the rest of the file. Only windows is different because of some difficulties with gradle not handling kill signal properly causing the hotreload to be unable to be killed by `ctrl-c`.